src/Private/Responses.ps1

```diff
@@ -345,9 +346,19 @@ function Write-PodeAttachmentResponseInternal {
 
     # Check if the path exists
     if ($null -eq $pathInfo) {
-        # If not, set the response status to 404 Not Found
-        Set-PodeResponseStatus -Code 404
-        return
+        #if not exist try with to find with public Route if exist
+        $Path = Find-PodePublicRoute -Path $Path 
+        if ($Path) {
+            # only attach files from public/static-route directories when path is relative
+            $Path = Get-PodeRelativePath -Path $Path -JoinRoot
+            # Attempt to retrieve information about the path
+            $pathInfo = Get-Item -Path $Path -ErrorAction Continue
+        }
```

Comment:
This is just a repeat of what `Set-PodeResponseAttachment` already does, as it will already attempt to look in the /public folder as well. Other places that calls this function are specific to static routes as well, so this would actually be slightly dangerous if you only want a file from a static route, and then it wonders into the /public folder.

Reply:
Set-PodeResponseAttachment is the public wrapper for this function. 
the reason of that is that now multiple function are calling Write-PodeAttachmentResponseInternal avoiding redundant code

Reply:
Resolved on the File Browser feature